pom.xml

```diff
@@ -124,13 +124,6 @@
             <version>${elasticsearch.version}</version>
         </dependency>
 
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.12</version>
-            <scope>test</scope>
-        </dependency>
-
```

Comment:
This is for a sperate ticket, but it was small enough so I'm sneaking it in this PR. I didn't see any specific reason to use junit4 over the junit5 available in spring dependency so I just removed it.